installer.sh

```diff
@@ -17,7 +51,13 @@ cp g910-gkeys.service "$DEST"/g910-gkeys.service
 
 # configuration file - will not overwrite existing files.
 [[ -d "$CONFDIR" ]] ||  mkdir "$CONFDIR"
-rsync -a -v --ignore-existing config/* "$CONFDIR"
+cp -a -n "config/config.$KEYBOARD.json" "$CONFDIR"/config.json || true
 
 # enable and start service
-systemctl enable --now g910-gkeys.service
+if [[ $STARTSERVICE = y ]]; then
+    systemctl enable --now g910-gkeys.service
```

Comment:
Starting service should be disabled on default. User should only get what they requested by default and when a user types "installer.sh" it means he wants to install the program not necessary enable it.

Reply:
I disagree on this. *all* software are enabled by default.
"labda" user does not know systemd. Power users will be able to do differently, which look correct for me.